src/validation.rs

```diff
@@ -1904,6 +1899,7 @@ fn validate_distribution(
 
             let exported = context.libpython_exported_symbols.contains(&ext.init_fn);
 
+            #[allow(clippy::needless_bool, clippy::if_same_then_else)]
```

Comment:
I opted to allow the if statements to keep their current structure though it is redundant because I find it quite clear this way and am not compelled to change the style.